stream-chat-android-client/src/main/java/io/getstream/chat/android/client/experimental/plugin/handler/StateHandler.kt

```diff
@@ -0,0 +1,20 @@
+package io.getstream.chat.android.client.experimental.plugin.handler
+
+/**
+ * Handles the state of stateful classes in the SDK. It can be used to clear the state of objects when a certain event
+ * occurs.
+ */
+public interface StateHandler {
```

Comment:
Isn't it possible to move this class to OfflinePlugin? It is not used in this module I believe.

Reply:
Removed StateHandler.